CMakeLists.txt

```diff
@@ -425,15 +427,15 @@ else()
       $<$<CXX_COMPILER_ID:GNU>:
         $<$<VERSION_GREATER:$<CXX_COMPILER_VERSION>,4.4>:
           -Wunused-result
-          -Wunused-parameter 
+          -Wunused-parameter
```

Comment:
Looks like a lot of extra white spaces in this file.

Reply:
Yeah, my formatter removed that automatically, I thought it'd be appreciated, if you prefer I revert the changes and leave the extra whitespace just say and I'll do it.

Reply:
Yeah, It just makes the review harder since there is a lot of other changes.